release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3367,4 +3367,42 @@ $ oc adm release info 4.13.18 --pullspecs
 [id="ocp-4-13-18-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-19"]
+=== RHSA-2023:6130 - {product-title} 4.13.19 bug fix and security update
+
+Issued: 2023-10-30
+
+{product-title} release 4.13.19, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:6130[RHSA-2023:6130] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:6133[RHBA-2023:6133] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.19 --pullspecs
+----
+
+[id="ocp-4-13-19-features"]
+==== Features
+
+[id="ocp-4-13-19-APIServer-insights-operator"]
+===== APIServer.config.openshift.io is now tracked by Insights Operator
+
+After running the Insights Operator, a new file is now available in the archive in the path `config/apiserver.json` with the information about the audit profile for APIServer.config.openshift.io.
++
+Access to audit profiles help you to understand what audit policy is common practice, what profiles are most commonly used, what differences there are between industries, and what kind of customization is applied.
+
+[id="ocp-4-13-19-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Cluster Version Operator did not reconcile SecurityContextConstraints (SCC) resources as expected. Cluster Version Operator (CVO) now properly reconciles the Volumes field SecurityContextConstraints resources towards the state defined in the release image. User modifications to system SCC resources are tolerated.
+
```

Comment:
Needs a `+` continuation between these two paragraphs to keep them together, since it is within a bulleted list item.